core/src/storage/recovery.py

```diff
@@ -10,13 +10,13 @@
 _IN_PROGRESS               = const(0x00)  # bool
 _DRY_RUN                   = const(0x01)  # bool
 _SLIP39_IDENTIFIER         = const(0x03)  # bytes
-_SLIP39_THRESHOLD          = const(0x04)  # int
```

Comment:
what happened here?

Reply:
This is not used, so I moved it under Deprecated Keys.

Looking at git history, it seems it hasn't been used from the start (or from moving code into this file)